packaging/installer/UPDATE.md

```diff
@@ -1,74 +1,158 @@
-under either  `/etc/cron.daily/`, or `/etc/periodic/daily/`. Whenever the `netdata-updater` is executed, it checks if a
-newer nightly build is available and then handles the download, installation and Netdata restart.  
+Docker-based installations do not update automatically. To update an agent running in a Docker container, you must pull
+the [latest image from Docker hub](https://hub.docker.com/r/netdata/netdata), stop and remove the container, and
+re-create it using the latest image.
 
-Note that after Jan 2019, the `kickstart.sh` one-liner `bash <(curl -Ss https://my-netdata.io/kickstart.sh)` calls the
-`netdata-installer.sh` with the auto-update option. So if you just run the one-liner without options once, your Netdata
-will be kept auto-updated.
+First, pull the latest version of the image.
 
-## You downloaded a binary package
+```bash
+docker pull netdata/netdata:latest
+```
+
+You will then need to stop and remove any containers using the `netdata/netdata` image. Replace `netdata` if you changed
+it from the default in our [Docker installation instructions](../docker/README.md#run-netdata-with-the-docker-command).
+
+```bash
+docker stop netdata
+docker rm netdata
+```
+
+You can now re-create your Netdata container using the `docker` command or a `docker-compose.yml` file. See our [Docker
+installation instructions](../docker/README.md#run-netdata-with-the-docker-command) for details. For example, using the
+`docker` command:
+
+```bash
+docker run -d --name=netdata \
+  -p 19999:19999 \
+  -v /etc/passwd:/host/etc/passwd:ro \
+  -v /etc/group:/host/etc/group:ro \
+  -v /proc:/host/proc:ro \
+  -v /sys:/host/sys:ro \
+  -v /etc/os-release:/host/etc/os-release:ro \
+  --cap-add SYS_PTRACE \
+  --security-opt apparmor=unconfined \
+  netdata/netdata
+```
 
-If you installed it from a binary package, the best way is to **obtain a newer copy** from the source you got it in the
-first place.
+## macOS
+
+If you installed Netdata on your macOS system using Homebrew, you can explictly request an update:
+
+```bash
+brew upgrade netdata
+```
+
+Homebrew will download the latest Netdata via the
+[formulae](https://github.com/Homebrew/homebrew-core/blob/master/Formula/netdata.rb), ensure all dependencies are met,
+and update Netdata via reinstallation.
+
+## Manual installation from Git
+
+If you installed Netdata manually from Git using `netdata-installer.sh`, you can run that installer again to update your
+agent. First, run our automatic requirements installer, which works on many Linux distributions, to ensure your system
+has the dependencies necessary for new features.
+
+```bash
+bash <(curl -sSL https://raw.githubusercontent.com/netdata/netdata/master/packaging/installer/install-required-packages.sh)
+```
+
+Then, navigate to the directory where you first cloned the Netdata repository, pull the latest source code, and run
+`netdata-install.sh` again. This process will compile Netdata with the latest source code and update it via
```

Comment:
```suggestion
`netdata-install.sh` again. This process compiles Netdata with the latest source code and update it via
```

Reply:
Fixed all these! Plus a few others. I was very into the future when I wrote this, apparently.